common/views/components/PrismicImage/PrismicImage.tsx

```diff
@@ -68,7 +67,7 @@ function createPrismicLoader(maxWidth: number) {
 
     searchParams.set(`auto`, 'compress,format');
     searchParams.set(`rect`, url.searchParams.get('rect') || '');
-    searchParams.set(`q`, (quality || 75).toString());
+    searchParams.set(`q`, (quality || 10).toString());
```

Comment:
Maybe wang this `defaultImageQuality` in a variable up top so it's easy to point at in future?